src/components/SignIn.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import InputForm from "./Form";
+
+function SignIn () {
```

Comment:
This looks pretty similar to the signup page, why don't you make them both into a single component called something like "Auth form" and have them take parameters for what they need.

Reply:
Addressed. Please review my updated codes. Thanks!